lab-jeff/lib/server0.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
Is this a duplicate file? Be sure to get rid of any unnecessary files before pushing up to your repo.